tests/skipCrossGenFiles.arm.txt

```diff
@@ -0,0 +1,2 @@
+mscorlib.dll
+System.Runtime.WindowsRuntime.UI.Xaml.dll
```

Comment:
This test fails because there is a missing *.dll.  It seems logical to believe the managed assembly is for windows only.